sdks/python/apache_beam/examples/complete/game/game_stats.py

```diff
@@ -111,6 +111,8 @@ def __init__(self):
 
   def process(self, elem):
     try:
+      if isinstance(elem, bytes):
```

Comment:
Can we always decode here? It is better to have a clear expectation of input arguments as much as possible on Python 3: either always encoded bytes, or always strings, but not mixing the two.